Assets/MixedRealityToolkit-Examples/StandardShader/Scenes/MaterialGallery.unity

```diff
@@ -0,0 +1,5793 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
```

Comment:
In immersive, the scene are not visible as the floor boundary blocks view.  Suggest either re-orientating the scene objects above the floor, or disabling floor in the boundary for this example.

Reply:
Good catch, I'll address this too!

Reply:
@DDReaper Sounds great. In general, the example scene that supports both HoloLens and IHMD should not have a table. Scene content adjustor script does not handle HL vs IHMD case which makes it harder to adjust content height for both devices. I removed the table in the InteractableObjectExample scene which was not a necessary element.

Reply:
Soo close.
![image](https://user-images.githubusercontent.com/1793042/36859953-f5a3ec64-1d77-11e8-9711-18ed94d4cad1.png)
As you can see from this screenshot, there are still a few issues:

1: The dialog window is unreadable when you start the scene.  Might be better to place it at an angle to the left or right of the camera start position.  Just so the viewer can read about what they are going to see before walking around. (shouldn't have to walk to the other end of the room to find out)

2: Need to position the elements down slightly, so all the headers can be seen without blocking each other (a minor point but just to improve its initial look)

Reply:
Great suggestions! Just made those changes and pushed them up.

Reply:
Great stuff, if you want to pull those conflicts from Dev_Working and resolve, should get this merged today, syncing and reviewing